pkg/processor/native/processor_test.go

```diff
@@ -138,6 +139,36 @@ func (s *BildProcessorSuite) TestBildProcessor_Watermark() {
 	assert.NotEqual(s.T(), s.srcData, output)
 }
 
+func (s *BildProcessorSuite) TestBildProcessor_FixOrientation() {
+	var testFiles = []string{
+		"./_testdata/exif_orientation/f2t.jpg",
+		"./_testdata/exif_orientation/f3t.jpg",
+		"./_testdata/exif_orientation/f4t.jpg",
+		"./_testdata/exif_orientation/f5t.jpg",
+		"./_testdata/exif_orientation/f6t.jpg",
+		"./_testdata/exif_orientation/f7t.jpg",
+		"./_testdata/exif_orientation/f8t.jpg",
+	}
+	expected, err := ioutil.ReadFile("./_testdata/exif_orientation/expected.jpg")
+	if err != nil {
+		panic(err)
+	}
+	for _, testFile := range testFiles {
+		file, err := ioutil.ReadFile(testFile)
+		if err != nil {
+			panic(err)
+		}
+		orientation, _ := GetOrientation(bytes.NewReader(file))
+		img, _, err := s.processor.Decode(file)
+		img = s.processor.FixOrientation(img, orientation)
+		actual, err := s.processor.Encode(img, "jpg")
+		if err != nil {
```

Comment:
at this part, instead of doing a conditional check and do panic in case of error happens, can't we just assert err to be nil? any particular reason why we are doing this? also, on other test scenarios, we don't do panic on missing test file, which in my opinion we should do.

Reply:
agreed, will add panic on missing test file on another PR then